src/DssSpell.sol

```diff
@@ -45,28 +59,52 @@ contract DssSpellAction is DssAction, DssSpellCollateralAction {
     // A table of rates can be found at
     //    https://ipfs.io/ipfs/QmVp4mhhbwWGTfbh2BzwQB9eiBrQBKiqcPRZCaAxNUaar6
     //
+
     // --- Rates ---
+    // uint256 internal constant ONE_FIVE_PCT_RATE = 1000000000472114805215157978;
 
     // --- Math ---
-    uint256 internal constant WAD = 10 ** 18;
+    // uint256 internal constant WAD = 10 ** 18;
+
+    address internal constant NEW_STARKNET_DAI_BRIDGE = 0x9F96fE0633eE838D0298E8b8980E6716bE81388d;
```

Comment:
Wards:

L1EscrowMom: `0xc238e3d63dfd677fa0fa9985576f0945c581a266`
ESM: `0x09e05ff6142f2f9de8b6b65855a1d56b6cfe4c58`
DSPauseProxy: `0xbe8e3e3618f7474f8cb1d074a26affef007e98fb`

Reply:
Correct if you mean wards of starknet escrow.

Reply:
This is the Dai Bridge that it's authed on.

Reply:
```
$ seth call 0x9F96fE0633eE838D0298E8b8980E6716bE81388d "wards(address)(uint256)" 0xc238e3d63dfd677fa0fa9985576f0945c581a266

>>> 1
```